static/app/components/forms/selectControl.tsx

```diff
@@ -55,6 +56,10 @@ const MultiValueRemove = (
   </selectComponents.MultiValueRemove>
 );
 
+const LoadingIndicator = () => (
```

Comment:
can we maybe name it `SelectControlLoadingIndicator`? or move the code to the object?

Reply:
yep